.github/workflows/release-gh-notes.yml

```diff
@@ -0,0 +1,46 @@
+name: release-gh-notes
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        description: 'Specify the version for this release'
+        type: string
+      npm_package:
+        required: true
+        description: 'npm package of the release'
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: install npm package
+        run: npm install ${{ github.event.inputs.npm_package }}@${{ github.event.inputs.version }}
+        shell: bash
+       - name: Get manifest file
+         id: manifest
+         run: echo ::set-output name=manifest::$(cat node_modules/${{ github.event.inputs.npm_package }}/internal/manifest.json)
+         shell: bash
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ fromJson(steps.manifest.outputs.manifest).commit }}
+          fetch-depth: 0
+      - name: Replace version in package.json
+        run: |
+          package_json="$(jq '.version = "${{ github.event.inputs.version }}"' package.json)" && \
+          echo -E "${package_json}" > package.json    
+      - name: Generate changelog
+        run: npx conventional-changelog-cli@2 -i CHANGELOG.md -s -p conventionalcommits
```

Comment:
There still can be unexpected updates in transitive dependencies breaking stuff.

But I do not want to over-engineer it for now